### ISSUE TITLE
Add benchmarks to the shrinking challenge

### ISSUE THREAD

**Issue by jackfirth** [DRMacIver/minithesis#7]
I was looking at https://github.com/jlink/shrinking-challenge and noticed it has benchmarks for Hypothesis. I think it would be handy to have benchmarks for Minithesis too, so it's easy to see how big of a difference the complexity of Hypothesis's shrinker makes when compared to the simpler (but easier to port) Minithesis shrinker.

**Comment by DRMacIver** [comment]
Yeah, absolutely. I'm keen for this to exist, but am a bit short on spoons at the moment so haven't been prioritising shrinking challenge work.

**Comment by DRMacIver** [comment]
Ideally it would be possible to make this work by just reusing the code from the existing Hypothesis shrinking challenges, but I'm not sure if that's easier than maintaining a separate version of each.

**Comment by jackfirth** [comment]
Separate versions sounds simplest to me, but I have _very_ little knowledge about how that repository is set up so I'm just shooting in the dark.
